Dockerfile

```diff
@@ -21,10 +21,9 @@ RUN apt-get -q update && \
     && rm -r /var/lib/apt/lists/*
 
 # Everything up to here should cache nicely between Swift versions, assuming dev dependencies change little
-ENV SWIFT_BRANCH=swift-3.0.2-release \
-    SWIFT_VERSION=swift-3.0.2-RELEASE \
-    SWIFT_PLATFORM=ubuntu16.04 \
-    PATH=/usr/bin:$PATH
```

Comment:
I'm not sure why it was there to begin with - maybe a relic of earlier iterations of this file. `/usr/bin` is already on the PATH from the Ubuntu base image:

    docker run -it ubuntu:16.04 env | grep PATH
    PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin